src/cascadia/TerminalSettingsModel/AllShortcutActions.h

```diff
@@ -106,7 +107,7 @@
     ON_ALL_ACTIONS(ShowContextMenu)         \
     ON_ALL_ACTIONS(ExpandSelectionToWord)   \
     ON_ALL_ACTIONS(CloseOtherPanes)         \
-    ON_ALL_ACTIONS(RestartConnection)
+    ON_ALL_ACTIONS(RestartConnection)       

```

Comment:
ah, I bet it's complaining about the trailing spaces here. 


```
tools\runformat.cmd
```

should automatically reformat everything for you 😉

Reply:
![image](https://github.com/microsoft/terminal/assets/15957528/5a8cecb0-bb48-49fb-96d4-be50afe69e4b)

Reply:
> ah, I bet it's complaining about the trailing spaces here.
> 
> ```
> tools\runformat.cmd
> ```
> 
> should automatically reformat everything for you 😉

Things are never easy are they. 
How can I solve. Noob at ps1 scripts but It seems like it should work cause the Invoke-CodeFormat function is inside the OpenConsole.psm1 script. As for now I went the manual route. Problem seems to be in ControlCore.cpp

Reply:
Sorry about that.

Can you do this instead? It lets you do the steps from "runformat" manually.

It will be much easier than trying all your individual files to see which one was the problem.

```
import-module ./tools/openconsole.psm1
invoke-codeformat -ignorexaml
```

it will make changes to your code which you will need to commit. :smile:

Reply:
@zadjii-msft @DHowett 

I solved the issues. It's ready to be reviewed now.